libs/providers/vmware.py

```
@@ -802,12 +802,20 @@ def get_obj(self, vimtype: Any, name: str) -> Any:
         finally:
             container.Destroy()
 
-    def add_rdm_disk_to_vm(self, vm: vim.VirtualMachine, rdm_type: Literal["virtual", "physical"]) -> None:
+    def add_rdm_disk_to_vm(
+        self,
+        vm: vim.VirtualMachine,
+        rdm_type: Literal["virtual", "physical"],
+        enable_cbt: bool = False,
```

Ruff (0.15.14)

[warning] Boolean-typed positional argument in function definition

(FBT001)